plugins/GSdx/GSRendererDX.cpp

```diff
@@ -23,10 +23,14 @@
 #include "GSRendererDX.h"
 #include "GSDeviceDX.h"
 
+bool s_IS_DX1011 = false;
```

Comment:
You should make the variable local to the class and use the `m_` prefix instead.